src/framework/mocha/Utilities/MOFunctionArgument.m

```diff
@@ -1095,4 +1095,55 @@ + (NSInteger)structureToJSValue:(JSValueRef *)value inContext:(JSContextRef)ctx
     return c - c0 - 1;
 }
 
++ (BOOL)isHighSierraOrHigher {
```

Comment:
Yep, CocoaScript was written by Gus and is entirely independent, so we wouldn’t want to introduce any dependencies on our stuff.